packages/amplify-provider-awscloudformation/src/index.ts

```diff
@@ -17,45 +54,20 @@ const { loadResourceParameters, saveResourceParameters } = require('./resourcePa
 const { formUserAgentParam } = require('./aws-utils/user-agent');
 const predictionsRegionMap = require('./aws-predictions-regions');
 
-import { adminLoginFlow } from './admin-login';
-import { adminBackendMap, isAmplifyAdminApp } from './utils/admin-helpers';
-import { CognitoUserPoolService, createCognitoUserPoolService } from './aws-utils/CognitoUserPoolService';
-import { IdentityPoolService, createIdentityPoolService } from './aws-utils/IdentityPoolService';
-import { S3Service, createS3Service } from './aws-utils/S3Service';
-import { DynamoDBService, createDynamoDBService } from './aws-utils/DynamoDBService';
-import { resolveAppId } from './utils/resolve-appId';
-import { loadConfigurationForEnv } from './configuration-manager';
-import { getLocationSupportedRegion, getLocationRegionMapping } from './aws-utils/aws-location';
-import { SSM } from './aws-utils/aws-ssm';
-import { Lambda } from './aws-utils/aws-lambda';
-import CloudFormation from './aws-utils/aws-cfn';
-import { $TSContext, ApiCategoryFacade } from 'amplify-cli-core';
-import * as resourceExport from './export-resources';
-import * as exportUpdateMeta from './export-update-amplify-meta';
-
 export { resolveAppId } from './utils/resolve-appId';
 export { loadConfigurationForEnv } from './configuration-manager';
 export { getLocationSupportedRegion, getLocationRegionMapping } from './aws-utils/aws-location';
-import { updateEnv } from './update-env';
 
 export const cfnRootStackFileName = 'root-cloudformation-stack.json';
 export { storeRootStackTemplate } from './initializer';
-import { transformResourceWithOverrides } from './override-manager';
 export { transformResourceWithOverrides } from './override-manager';
-import { rootStackFileName } from './push-resources';
 export { rootStackFileName } from './push-resources';
 
-import { compileSchema } from './utility-functions';
-import { LocationService } from './aws-utils/aws-location-service';
-import { hashDirectory } from './upload-appsync-files';
-import { prePushCfnTemplateModifier } from './pre-push-cfn-processor/pre-push-cfn-modifier';
-import { getApiKeyConfig } from './utils/api-key-helpers';
-
 function init(context) {
   return initializer.run(context);
 }
 
-function initEnv(context, providerMetadata) {
+export function initEnv(context, providerMetadata) {
```

Comment:
these functions were already exported using module.exports below but to get proper TS compilation they need to be exported as well